ui/imports/utils/ProfileUtils.qml

```diff
@@ -78,6 +78,16 @@ QtObject {
         return Constants.socialLinkType.custom
     }
 
+    function linkTypeToIcon(linkType) {
```

Comment:
Have you seen the icon also comes from NIM? With this new function, could we remove the code there? FYI: https://github.com/status-im/status-desktop/issues/13817

Reply:
yeah. The nim icons need to be removed